src/opera/commands/info.py

```diff
@@ -46,7 +53,7 @@ def _parser_callback(args):
 
     storage = Storage.create(args.instance_path)
     try:
-        outs = info(storage)
+        outs = info(args.csar_or_rootdir, storage)
         print(format_outputs(outs, args.format))
```

Comment:
If you rebase this branch onto master you will see that `format_outputs` are now available as a helper function from `utils.py`. So I propose (as you are already modifying this file) removing `format_outputs` from this file and importing the one from `utils.py` which you can then use.

Reply:
done!